app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +35,15 @@
                 android:name="android.app.lib_name"
                 android:value="" />
         </activity>
+
+        <receiver
+            android:name=".ActionBroadcastReceiver"
+            android:enabled="true"
+            android:exported="false">
```

Comment:
whats the purpose of these each flag?

Reply:
android:enabled="true - if the specified activity can be instantiated by the system or not, we don't need it here as default value is true
android:exported - if the specified activity can be lauched by apps other than android system.